src/test/java/com/epam/service/MockPoliceForceServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.service;
+
+import com.epam.service.MockPoliceForceService;
+import org.junit.Test;
+
+import java.util.concurrent.*;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Created by Alexey on 03.05.2016.
+ */
+public class MockPoliceForceServiceTest {
```

Comment:
i suggest you use Mockito to check positive(thenReturn) and negative (thenThrow) behaviour for your service.
Also you can validate if correct methods were called.